testprojects/maven_layout/junit_resource_collision/onedir/src/test/java/BUILD

```diff
@@ -2,6 +2,7 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 junit_tests(
+  name='test',
```

Comment:
This change doesn't look necessary?

Reply:
Unfortunately it's needed to avoid https://github.com/google/error-prone/pull/1007